### ISSUE TITLE
Implicit multiplication and logarithm

### ISSUE THREAD

**Issue by jullienaline** [numworks/epsilon#1559]
#### Describe the bug
By writing xln(2) without the multiplication sign, the numworks misinterprets the position of the product.
There is no problem writing ln(2)x or 5ln(2).

#### Screenshots
![image](https://user-images.githubusercontent.com/65825309/82736540-c820c700-9d2a-11ea-8e58-9c713f3d7160.png)

#### To Reproduce
1. Go to the 'Calculation' app
2. Type 'x sto 2'
3. Type 'xln(2)'

#### Expected behavior
result : x.ln(2)
              

#### Environment
 - Epsilon version 13.2.0
 - online simulator, actual device

**Comment by RedGl0w** [comment]
This is related to #1071
The issue is that `ln()` isn't like on other calculators a specific element, but only a string. The design of poincare, when seeing so `xln(` will think that `xln` is a function.
I don't think this issue will be patched, because it would need a lot of changes in poincare

**Comment by suazhero** [comment]
Hello, was indeed clearly stated that it's by design.
Still think it can be highly confusing for high school students.
Especially since they may not understand why "multiply" can sometimes be ellided and sometimes not (although the explanation makes sense).
For my daughter, I worked around this telling to input "multiply" any time she means to multiply something (even 5x for instance).

This, with the auto zoom in functions app are the 2 things that still bother me in epsilon (for the rest it's close to being perfect).

**Comment by Ecco** [comment]
@suazhero Do you have any idea on how we could make this better then? If we accept `xln(x)` as meaning `x*ln(x)`, then why wouldn't `acos(x)` mean `a*cos(x)`? Or why wouldn't `xln(x)` mean `xl*n(x)`?

**Comment by suazhero** [comment]
No, considering the way it's done it's tricky.
And it's true it would have to be the same for cos, sin etc etc...
Apart from reserving these expressions and interpret them as a whole instead of a string like any other i don't see how.

**Comment by RedGl0w** [comment]
Yes, the only solution is to interpret them as element, and not as string, but this could create some issues for simulator (because clicking on the virtual keyboard is longer than writing `cos(`, even if there is keyboard shortcut (because they're unknown by a lot of people).

**Comment by suazhero** [comment]
Or, you could catch them when terminating a string right before parenthesis (. Not sure how dirty this is. And would prevent to use custom functions such as customln(x) wchich would become custom*ln(x)

**Comment by RedGl0w** [comment]
By the way, an article on numworks.com has been made about this issue : https://www.numworks.com/blog/make-functions-accessible-from-all-applications/

**Comment by artaxxx** [comment]
Fixed in 19.0.0
